Fitness/Controllers/ClassDetailViewController.swift

```diff
@@ -395,8 +395,10 @@ class ClassDetailViewController: UIViewController, UITableViewDelegate, UITableV
     @objc func favorite(){
         print("favorite")
```

Comment:
Why do we do this? Let’s remove this print statement and just make this one line `@objc func favorite() {}`